### PR TITLE
Adds a Dockerfile

### PR DESCRIPTION
**Pull request by PotHix**
It is now possible to run Simplestack on a Docker container.

### CODE REVIEW

**Comment by PotHix** [comment]
@rafaellima

**Comment by bacarini** [comment]
👏🏼👏🏼👏🏼👏🏼

**Comment by PotHix** [comment]
Just in development mode for now, but it is a good step :)

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,22 @@
+FROM debian:jessie
+MAINTAINER PotHix
+RUN apt-get update
```
Comment (rafaellima) [inline Dockerfile]: Only one issue here. Running `apt-get update` alone in a statement causes caching issues. It think we'll be better off with something like this:

```
RUN apt-get update && apt-get install -y --force-yes /
```

**Comment by rafaellima** [comment]
:+1:

**Pull request merged**